chainstate/test-suite/src/tests/reorgs_tests.rs

```diff
@@ -81,7 +81,12 @@ fn reorg_simple(#[case] seed: Seed) {
 fn test_very_long_reorgs(#[case] seed: Seed) {
     utils::concurrency::model(move || {
         let mut rng = make_seedable_rng(seed);
-        let mut tf = TestFramework::default();
+        let mut tf = TestFramework::builder()
+            .with_chainstate_config(chainstate::ChainstateConfig {
+                tx_index_enabled: true.into(),
+                ..Default::default()
+            })
```

Comment:
I think for TestFramework we should make `true` the default value. It will increase coverage.

Also I'd suggest adding another case to `tx_verification_simulation.rs`. So that it runs with and without tx index.

Reply:
> I think for TestFramework we should make true the default value. It will increase coverage.

Good idea. Fixed, thanks.

> Also I'd suggest adding another case to tx_verification_simulation.rs. So that simulation runs with and without tx index.

Added, thanks.